jina/optimizers/parameters.py

```diff
@@ -37,38 +44,36 @@ def __init__(
         self.log = log
```

Comment:
Why do I need to use `log` when optimizing `IntegerParameters`?

Reply:
If you want a logarithmic scale instead of a uniform sampling via step-size. And here you got me: We are not consistent about how we create parameters and sampling.